packages/tradershub/src/screens/PersonalDetails/Sections/TaxInfoConfirmation.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Field, FieldProps } from 'formik';
+import { Checkbox } from '@deriv-com/ui';
+
+const TaxInfoConfirmation = () => {
+    return (
+        <Field name='taxInfoConfirmation' type='checkbox'>
+            {({ field, meta }: FieldProps) => (
+                <Checkbox
+                    error={Boolean(meta.error && meta.touched)}
+                    id='taxInfoConfirmation'
+                    label='I confirm that my tax information is accurate and complete.'
+                    {...field}
+                />
+            )}
+        </Field>
+    );
+};
```

Comment:
you can remove the return syntax and curly braces and just enclosed it with a normal bracket for a shorter syntax